src/main/java/org/jabref/logic/remote/client/RemoteClient.java

```diff
@@ -52,7 +54,13 @@ public boolean ping() {
      */
     public boolean sendCommandLineArguments(String[] args) {
         try (Protocol protocol = openNewConnection()) {
-            protocol.sendMessage(RemoteMessage.SEND_COMMAND_LINE_ARGUMENTS, args);
+
+            String[] encodedArgs = args.clone();
```

Comment:
Can you please put this addition in the `protocol.sendMessage` so that only the protocol has to worry about decoding/encoding. Also please add a short comment explaining why we need to encode. Thanks!